msal4j-sdk/src/main/java/com/microsoft/aad/msal4j/TokenCache.java

```diff
@@ -346,7 +346,7 @@ Set<IAccount> getAccounts(String clientId) {
                         ((Account) rootAccounts.get(accCached.homeAccountId())).tenantProfiles.put(accCached.realm(), profile);
                     }
 
-                    if (accCached.homeAccountId().contains(accCached.localAccountId())) {
+                    if (accCached.localAccountId() != null && accCached.homeAccountId().contains(accCached.localAccountId())) {
```

Comment:
Seems like a separate change?

Reply:
This PR was meant to handle changes for all the small issues/enhancements we've gotten over the last month, but since the PRs made by customers didn't need additional changes this PR ended up handling just the timeout behavior and the NPE exception reported in https://github.com/AzureAD/microsoft-authentication-library-for-java/issues/669

Since it was just a one-line change, I figured I'd just add it into this PR anyway.